src/js/lib/content/src/index.html

```diff
@@ -43,6 +40,9 @@
                 -webkit-app-region  : drag;
                 -webkit-user-select : none;
             }
+            .mapboxgl-map {
+                border-radius: 14px;
+            }
```

Comment:
Could it be somehow added by loaded visualization itself, instead of embedding it into index.html?